exec/jdbc-all/pom.xml

```diff
@@ -441,7 +441,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
                   
                   </message>
-                  <maxsize>20000000</maxsize>
+                  <maxsize>21000000</maxsize>
```

Comment:
Once I have added that:
<groupId>org.eclipse.jetty</groupId>
<artifactId>jetty-servlets</artifactId>
<version>9.3.9.v20160517</version>

size grew, I will change version to match with jetty version.

Maybe we should add exclusions for that dependency. 
@hnfgns Can you suggest what to add in exclusions list?